src/JsonValidate.JSInterop/wwwroot/codemirror.js

```diff
@@ -18,11 +17,12 @@ window.codemirror = {
     },
     setValue: function (value) {
         cm.doc.setValue(value);
-        return null;
     },
-    highlight: function (fromLine, fromChar, toLine, toChar)
-    {
-        cm.doc.markText({line: fromLine, ch: fromChar}, {line: toLine, ch: toChar}, {
+    setOption: function (key, value) {
+        cm.setOption(key, value);
+    },
+    highlight: function (fromLine, fromChar, toLine, toChar) {
+        cm.doc.markText({ line: fromLine, ch: fromChar }, { line: toLine, ch: toChar }, {
             css: "background: #81dee4"
```

Comment:
This colour doesn't work in dark mode:

<img width="435" alt="Screen Shot 2020-05-08 at 8 46 40 am" src="https://user-images.githubusercontent.com/24448509/81351828-8bd55000-9108-11ea-9640-90b1713f9d0e.png">

Could you update the function to take in the current theme (e.g. `isDark`) and update the colour to #7E211B when in dark mode, otherwise set as #81dee4?

Reply:
I can do that, good catch!